kinto/tests/test_views_collections_cache.py

```diff
@@ -0,0 +1,81 @@
+from .support import (BaseWebTest, unittest, MINIMALIST_RECORD)
+
+
+class ForcedCacheExpiresTest(BaseWebTest, unittest.TestCase):
```

Comment:
Ok thanks I will clarify this. (one tests `cache_expires` from settings, the other one from json api)